controller/test/testmanager.spec.ts

```diff
@@ -345,36 +347,60 @@ describe("TestManager", () => {
             throw error;
           }
         }));
+        // Tags 
+        validateLegacyOnlySuite.addTest(new MochaTest(version + " should return tags for that version", async () => {
+          try {
+            mockGetObjectTagging(new Map([[VERSION_TAG_NAME, version]]));
+            await getValidateLegacyOnly(version).then((result: boolean | undefined) => {
+              log("getValidateLegacyOnly()", LogLevel.INFO, result);
+              expect(result).to.equal(expected)
+            });
+          } catch (error) {
+            throw error; 
+          }
+        }))
       }
     });
 
+    after(() => {
+      resetMockS3();
+    });    
+
     it("should return undefined for undefined", async () => {
       try {
-        await getValidateLegacyOnly(latestPewPewVersion).then((result: boolean | undefined) => {
-          expect(result).to.equal(undefined);
-        });
+        const expected = "";
```

Comment:
expected should be `undefined`, not empty string. Empty string should never be retunred.